tests/generic_config_updater/test_portchannel_interface.py

```diff
@@ -264,7 +264,12 @@ def verify_attr_change(duthost, po_name, attr, value):
             ...
     """
     if attr == "mtu":
-        output = duthost.shell("show interfaces status | grep -w '^{}' | awk '{{print $4}}'".format(po_name))
+        cmd = (
+            "show interfaces status | "
+            "grep -w '^[[:space:]]*{}' | "
+            "awk '{{print $4}}'"
+        ).format(po_name)
+        output = duthost.shell(cmd)
```

Comment:
we have `show_and_parse` helper function in `sonic.py`, is it better to leverage this common function in our script instead of parsing the output by itself?